pkg/virtctl/vm/add_volume_test.go

```diff
@@ -57,41 +57,16 @@ var _ = Describe("Add volume command", func() {
 		coreClient = fake.NewSimpleClientset()
 	})
 
-	createTestDataVolume := func() *v1beta1.DataVolume {
-		return &v1beta1.DataVolume{
-			ObjectMeta: k8smetav1.ObjectMeta{
-				Name: "testvolume",
-			},
-		}
-	}
-
-	createTestPVC := func() *k8sv1.PersistentVolumeClaim {
-		return &k8sv1.PersistentVolumeClaim{
-			ObjectMeta: k8smetav1.ObjectMeta{
-				Name: "testvolume",
-			},
-		}
-	}
-
-	verifyVolumeSource := func(volumeOptions interface{}, useDv bool) {
-		if useDv {
-			Expect(volumeOptions.(*v1.AddVolumeOptions).VolumeSource.DataVolume).ToNot(BeNil())
-			Expect(volumeOptions.(*v1.AddVolumeOptions).VolumeSource.PersistentVolumeClaim).To(BeNil())
-		} else {
-			Expect(volumeOptions.(*v1.AddVolumeOptions).VolumeSource.DataVolume).To(BeNil())
-			Expect(volumeOptions.(*v1.AddVolumeOptions).VolumeSource.PersistentVolumeClaim).ToNot(BeNil())
-		}
-	}
-
 	expectVMIEndpointAddVolume := func(vmiName, volumeName string, useDv bool) {
 		kubecli.MockKubevirtClientInstance.
 			EXPECT().
 			VirtualMachineInstance(k8smetav1.NamespaceDefault).
 			Return(vmiInterface).
 			Times(1)
 		vmiInterface.EXPECT().AddVolume(context.Background(), vmiName, gomock.Any()).DoAndReturn(func(ctx context.Context, arg0, arg1 interface{}) interface{} {
-			Expect(arg1.(*v1.AddVolumeOptions).Name).To(Equal(volumeName))
-			verifyVolumeSource(arg1, useDv)
+			volumeOptions := arg1.(*v1.AddVolumeOptions)
```

Comment:
Validate that type assertion was ok?
```suggestion
			volumeOptions, ok := arg1.(*v1.AddVolumeOptions)
			Expect(ok).To(BeTrue())
```

Reply:
i think that it cause panic if it is not ok (saw it today when i forgot a ptr on the cast on other place)
anyhow the purpose of this PR is mostly to remove the inline, unless you insist